actix-cors/src/lib.rs

```diff
@@ -1061,6 +1061,7 @@ mod tests {
         );
 
         {
+            #[allow(clippy::needless_collect)]
```

Comment:
This seems not to suppress the warning, according to the annotation?

Reply:
Ahh... thanks. You'd think the "needless collect" annotation should go on the expression where the collection actually happens.